easybuild/easyconfigs/b/BOLT-LMM/BOLT-LMM-fix-makefile.patch

```diff
@@ -0,0 +1,38 @@
+diff -ru BOLT-LMM_v2.3.4.orig/src/Makefile BOLT-LMM_v2.3.4/src/Makefile
```

Comment:
@btlindert please add a comment here explaining the purpose of the patch.

Reply:
The patch updates the build arguments in the Makefile with are orginally for Intel. These can also be added to "buildopts" though, if preferred.

Reply:
That's fine - please add the explanation as a comment inside the patch file.